restli-server/src/main/java/com/linkedin/restli/internal/server/response/CollectionResponseBuilder.java

```diff
@@ -166,8 +172,13 @@ private D buildRestLiResponseData(final Request request,
     final AnyRecord projectedCustomMetadata;
     if (customMetadata != null)
     {
+      DataMap customData = customMetadata.data();
```

Comment:
nit: this variable name is misleading. I'd rename to `customMetadataMap` or `customMetadataData` to be consistent with the variable `projectedCustomMetadata` below.

Reply:
Changed that in next revision